nyc_taxi/anaconda-project.yml

```diff
@@ -14,26 +14,26 @@ examples_config:
 user_fields: [examples_config]
 
 channels:
-- pyviz
+- pyviz/label/dev
 - defaults
 
 packages: &pkgs
-- python=3.8
-- notebook<7
-- ipykernel=5.3.4
-- bokeh<2.4
-- datashader=0.12.1
-- fastparquet=0.5.0
-- holoviews=1.14.4
-- numpy<1.21
-- pandas<1.4
-- panel=0.11.3
-- python-snappy=0.6.0
-- tornado=6.1
-- xarray=0.18
-- numba<0.55
-- dask<2.11
-- jinja2<3
+- python=3.11
+- notebook
+- ipykernel
+- bokeh>=3.3.4
+- datashader
+- fastparquet
+- holoviews>=1.19.0a0
+- hvplot
+- numpy
+- pandas
+- panel
+- python-snappy
+- tornado
+- xarray
+- numba
```

Comment:
Same comment for `xarray` and `numba`. They are not imported directly in the notebooks, and they are dependencies of `datashader` that is also listed.

Reply:
Same comment as for `tornado`.